### DISCUSSION TITLE
Implement Warpcast Link Support

### DISCUSSION THREAD

**Discussion by dalechyn** [community/community#112633] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Warpcast is a client for Farcaster, a new type of social network. It's decentralized, like email, which means you control your account. It's an ever-growing community of interesting, curious people from all over the world.

DAU on Farcaster is currently around 25k users (https://dune.com/pixelhack/farcaster), and it’s like Twitter in the old days – but better.

There are several figures in Web3 engineering space that also have GitHub accounts and it would be awesome to have Warpcast icon and username instead of the long link. You can find it on my profile page.

**Comment by pjpscriv** [comment]
Hey @dalechyn, I've got a [little browser extension](https://github.com/pjpscriv/github-social-icons) that does this, and just added support for Warpcast / Farcaster in v1.1.0. You can check it out [here](https://chromewebstore.google.com/detail/github-social-icons/ledidlccnpfooopagmkgihgibfbnbbnp) (once the new version's been approved by the Chrome Webstore).

<img width="305" height="82" alt="image" src="https://github.com/user-attachments/assets/996e926d-2b3c-452f-b751-e0c71f377e01" />
